code401challenges/src/test/java/linkedList/LinkedListTest.java

```diff
@@ -95,4 +95,17 @@ public void insertAfterTest() {
 
         assertEquals(expected, actual);
     }
+
```

Comment:
this is only one test for the kth from the end method. i would like to see more tests, get some edge cases in there, to the same thing a couple times just slightly different. anything that shows that it works all the time, not just this one time